tooling/cli/lib/file-generators.js

```diff
@@ -14,13 +15,9 @@ const initProject = async (dir, instances) => {
   return generateConfigFile(dir, instances);
 };
 
-const generateEnvFile = async (dir, instances) => {
-  const fileName = path.resolve(dir, '.env');
-  let data = '';
-
-  instances.forEach(instance => {
-    data += `TI_${instance.nickname}_API_KEY=${instance.apiKey}\n`;
-  });
```

Comment:
Did we intentionally lose this?

Reply:
Yeah -- it's leftover from when we were going to store all the instance info in a `.env` but opted for `ti-config.json` instead. It's never actually referenced anywhere.